### PR TITLE
Implement glean metrics ping

### PR DESCRIPTION
**Pull request by travis79**
Adding schema for glean metrics ping

### CODE REVIEW

**Comment by travis79** [comment]
@mdboom, @fbertsch This is my first go at a pipeline schema PR, so please let me know if I messed anything up.

File: validation/org-mozilla-samples-glean/metrics.1.all.pass.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "moz://mozilla.org/schemas/glean/ping/1",
```
Comment (mdboom) [inline validation/org-mozilla-samples-glean/metrics.1.all.pass.json]:
```suggestion
  "$schema": "moz://mozilla.org/schemas/glean/ping/1",
```

File: validation/glean/metrics.1.all.pass.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "moz://mozilla.org/schemas/glean/ping/1",
```
Comment (mdboom) [inline validation/glean/metrics.1.all.pass.json]:
```suggestion
  "$schema": "moz://mozilla.org/schemas/glean/ping/1",
```

**Comment by travis79** [comment]
Still waiting on @fbertsch to take a look at this.  I don't have rights to merge so someone may need to take care of that once all reviews are done.  :)

**Pull request merged**